### PR TITLE
[GLUTEN-1575][VL] Add a parameter run_setup_script to control whether to run velox setup

### PR DESCRIPTION
**Pull request by liujiayi771**
## What changes were proposed in this pull request?
https://github.com/oap-project/gluten/issues/1575

### CODE REVIEW

**Comment by liujiayi771** [comment]
@zhejiangxiaomai Could you help review this PR?

File: ep/build-velox/src/build_velox.sh [hunk 1]
```diff
@@ -39,6 +40,10 @@ for arg in "$@"; do
     ENABLE_BENCHMARK=("${arg#*=}")
     shift # Remove argument name from processing
     ;;
+  --run_setup_script=*)
```
Comment (winningsix) [inline ep/build-velox/src/build_velox.sh:43]: @zhejiangxiaomai No directly related to this PR. As ```arg``` number grows, should we consider add a [help/usage method](https://opensource.com/article/19/12/help-bash-program)?

Comment (zhejiangxiaomai) [reply ep/build-velox/src/build_velox.sh:43]: Thanks, I can add it in next PR.

**Review by zhejiangxiaomai — COMMENTED** [review]
@liujiayi771 Thanks a lot, just one small suggestion. Can you add doc in https://github.com/oap-project/gluten/blob/main/docs/get-started/GlutenUsage.md ?

**Comment by liujiayi771** [comment]
> @liujiayi771 Thanks a lot, just one small suggestion. Can you add doc in https://github.com/oap-project/gluten/blob/main/docs/get-started/GlutenUsage.md ?

Done.

**Review by zhejiangxiaomai — APPROVED** [review]

**Pull request merged**